business/checkers/destinationrules/no_dest_checker.go

```diff
@@ -1,6 +1,8 @@
 package destinationrules
 
 import (
+	"github.com/kiali/kiali/config"
```

Comment:
as per coding standard, this should be moved to the kiali import section (the third section in our imports)